ckanext/data_qld_theme/templates/macros/data_qld_form.html

```diff
@@ -0,0 +1,15 @@
+{% import 'macros/form.html' as form %}
+
+{% macro select_width_disabled(name, id='', label='', options='', selected='', error='', classes=[], attrs={'class': 'form-control'}, is_required=false) %}
```

Comment:
Is this a typo `width`? should it be `with`?

Reply:
refactored